### ISSUE TITLE
After reboot, can't login via SSH => Permission denied (publickey)

### ISSUE THREAD

**Issue by redenm** [notthebee/ansible-easy-vpn#188]
- [x] I've read the [FAQ](https://github.com/notthebee/ansible-easy-vpn/blob/main/FAQ.md) and my problem isn't described there.
- [x] I've also searched through the existing issues, and my problem hasn't been brought up

**Describe the issue**

I just followed what's on the video tutorial.  Rebooted the server now I can't access via SSH.

Also, I can't access the UI.

I set the domain records as following:

A => vpn.domain.com => Points  to my server IP
A => auth.vpn.domain.com => Points  to my server IP
A => wg.vpn.domain.com => Points  to my server IP

**Logs (if applicable)** 
Paste any relevant error messages here:
```
Permission denied (publickey)
```

However, I can access via Lish. I'm using Linode.

**Environment:**
- Client OS: Ubuntu 22.04 WSL
- VPS OS: Ubuntu 22.04
- VPS Provider Linode

Just destroyed my server. I could not make it work. Also after reboot I got this error

```
ssh: connect to host [IP] port 22: Connection refused
```

Sometimes, it's permission denied

**Comment by notthebee** [comment]
Potential duplicate of #162

**Comment by redenm** [comment]
> Potential duplicate of #162

yeah almost the same but for me, I never accessed the web ui.

**Comment by Sega16tbit** [comment]
Same. Does not access web ui => check ports, change the domain name, run bash bootstrap.sh =>ssh: Permission denied (publickey)

**Comment by notthebee** [comment]
Should be fixed in v0.2
